src/awkward/_connect/_numba/arrayview.py

```diff
@@ -907,20 +907,22 @@ class type_method(numba.core.typing.templates.AbstractTemplate):
 
                     def generic(self, args, kwargs):
                         if len(kwargs) == 0:
-                            sig = typer(recordviewtype, args)
+                            sig = typer(recordviewtype, args)  # noqa: B023
                             sig = numba.core.typing.templates.Signature(
                                 sig.return_type, sig.args, recordviewtype
                             )
                             numba.extending.lower_builtin(
-                                methodname,
+                                methodname,  # noqa: B023
                                 recordviewtype,
                                 *[
                                     x.literal_type
                                     if isinstance(x, numba.types.Literal)
                                     else x
                                     for x in args
                                 ],
-                            )(lower)
+                            )(
+                                lower  # noqa: B023
+                            )
                             return sig
```

Comment:
Safe because `generic` is returned as soon as it's created; it never sees a different value of `args` and `methodname` than the ones it has at the time of its construction.